src/test/regress/expected/shard_rebalancer.out

```diff
@@ -2114,6 +2120,12 @@ WHERE logicalrelid = 'r1'::regclass;
      1
 (1 row)
 
+SELECT replicate_table_shards('t1',  shard_replication_factor := 2);
```

Comment:
Would it make more sense if `replicate_table_shards` would work with tables with streaming replication?

I do not think so

Reply:
no, probably not

Reply:
no, I don't think so. It would break metadata syncing.